### PR TITLE
fix msktutil url; add test to msktutil.

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/msktutil.rb [hunk 1]
```diff
@@ -19,4 +19,8 @@ def install
                           "--mandir=#{man}"
     system "make", "install"
   end
+
+  test do
+    system "#{sbin}/msktutil", "--version"
```
Comment (zmwangx) [inline Formula/msktutil.rb]: Usually this would be something like

```ruby
assert_match version.to_s, shell_output("#{sbin}/msktutil --version")
```

**Review by zmwangx — CHANGES_REQUESTED** [review]

File: Formula/msktutil.rb (hunk 1 above)
Comment (yurikoles) [inline Formula/msktutil.rb]: done

**Comment by yurikoles** [comment]
I had added devel.

File: Formula/msktutil.rb [hunk 2]
```diff
@@ -1,8 +1,16 @@
 class Msktutil < Formula
-  desc "Program for interoperability with Active Directory"
-  homepage "https://code.google.com/p/msktutil/"
-  url "https://msktutil.googlecode.com/files/msktutil-0.5.1.tar.bz2"
-  sha256 "ec02f7f19aa5600c5d20f327beaef88ee70211841dc01fa42eb258ae840ae6f0"
+  desc "Active Directory Keytab Management"
+  homepage "https://sourceforge.net/projects/msktutil/"
+
+  stable do
```
Comment (zmwangx) [inline Formula/msktutil.rb]: No need for `stable do` here, keep `url` and `sha256` where they used to be.

**Review by zmwangx — CHANGES_REQUESTED** [review]

File: Formula/msktutil.rb (hunk 2 above)
Comment (yurikoles) [inline Formula/msktutil.rb]: done

**Comment by yurikoles** [comment]
Up & rebase

**Comment by zmwangx** [comment]
I fixed a few lingering style issues and merged as 12cf50b. Thank you for your contribution to Homebrew!

**Comment by yurikoles** [comment]
@zmwangx thanks!

**Pull request closed without merging**